src/block/block_compact.c

```diff
@@ -403,12 +406,13 @@ __block_compact_estimate_remaining_work(WT_SESSION_IMPL *session, WT_BLOCK *bloc
 
     block->compact_estimated = true;
     block->compact_pages_rewritten_expected = block->compact_pages_rewritten + total_pages_to_move;
+    block->compact_bytes_rewritten_expected =
+      block->compact_pages_rewritten_expected * (uint64_t)avg_block_size;
 
     __wt_verbose_level(session, WT_VERB_COMPACT, verbose_level,
       "%s: expecting to move approx. %" PRIu64 " more pages (%" PRIu64 "MB), %" PRIu64
       " total, target %" PRIu64 "MB (%" PRIu64 "B)",
-      block->name, total_pages_to_move,
-      total_pages_to_move * (uint64_t)avg_block_size / WT_MEGABYTE,
+      block->name, total_pages_to_move, block->compact_bytes_rewritten_expected / WT_MEGABYTE,
```

Comment:
The code looks different now. Before it used `total_pages_to_move` which does not take into account the previous `block->compact_pages_rewritten`.

Reply:
Ah that's true, I'll change this back since the message is about how _more_ bytes are moved.

Reply:
https://github.com/wiredtiger/wiredtiger/pull/10354/commits/5f037aa5a80b6534e7c091ad925e7aa4f79dbe39